contracts/src/v0.8/KeeperRegistry.sol

```diff
@@ -209,7 +207,7 @@ contract KeeperRegistry is
     uint32 gasLimit,
     address admin,
     bytes calldata checkData
-  ) external override onlyOwnerOrRegistrar returns (uint256 id) {
+  ) external override onlyOwnerOrRegistrar whenNotPaused returns (uint256 id) {
```

Comment:
could we move this to `_createUpkeep` so that it can be used for migrations also?

Reply:
moved